activerecord/CHANGELOG.md

```diff
@@ -1229,4 +1229,16 @@
 
     *Yves Senn*
 
+*   Schema Migrations now contain time of creation information to track when
+    migrations were run.
+    ```
+	  database: vagrant_development
+
+	  Status   Migration ID    Migration Run             Migration Name
+	  ---------------------------------------------------------------------------
+	  up       20150918171109  2015-09-18 17:20:22 UTC   ********** NO FILE **********
```

Comment:
That is how the code currently works.  It notes migration files that are missing in, i guess, an alarming way.
I'm fine with the functionality -- didn't see a reason to change it.